build.psm1

```diff
@@ -462,6 +462,7 @@ Fix steps:
     # ARM is cross compiled, so we can't run pwsh to enumerate Experimental Features
```

Comment:
@TravisEz13, your last commit had 1 failures in `PowerShell-CI-windows`
Default enablement of Experimental Features.On preview builds, Experimental Features are enabled
```powershell
Expected: Feature PSDesiredStateConfiguration.InvokeDscResource to be Enabled
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ExperimentalFeature\Get-ExperimentalFeature.Tests.ps1: line 176
176:             $expFeature.Enabled | Should -BeEnabled -Name $expFeature.Name
```